tests/common_utils.py

```diff
@@ -0,0 +1,839 @@
+r"""Importing this file must **not** initialize CUDA context. test_distributed
```

Comment:
I don't think we need all this file and the next one (which comes from PyTorch I suspect?) here.
Can't we just do some basic functionality like in https://github.com/facebookresearch/maskrcnn-benchmark/blob/master/tests/test_data_samplers.py ?